layouts/shortcodes/notices/adminShip.html

```diff
@@ -0,0 +1 @@
+{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Admin-Only Ship" "noticeDescription" "This ship is classified as an Admin Ship and can only be used by Galaxy Staff.\nThis ship class is unobtainable and can only be given via the /addship command" "noticeColor" "#ffff00" ) ) }}
```

Comment:
Admin ships aren't given out with `/addship` anymore, they're just in a submenu of a staff member's inventory. Additionally, traditional `\n` newlines do not work in html files. You can use `<br />` instead, but I don't think a linebreak is necessary here anyway with my modifications to the actual wording.
```suggestion
{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Admin-Only Ship" "noticeDescription" "This ship is classified as an Admin Ship and can only be used by Galaxy Staff. Normal players cannot obtain this ship." "noticeColor" "#ffff00" ) ) }}
```